flow/layers/container_layer.h

```diff
@@ -38,6 +47,11 @@ class ContainerLayer : public Layer {
   // For OpacityLayer to restructure to have a single child.
   void ClearChildren() { layers_.clear(); }
 
+  void cacheTotalElevation();
```

Comment:
My understanding of the naming style is `ComputeFoo()` or `foo()` if there is a private member with name `foo_`. That's just what I inferred from our existing code. CC @chinmaygarde for the correct naming style.

Reply:
Updated to `ComputeTotalElevation`.